### PR TITLE
Update RIT AWS APPLY TERRAFORM formula README (#196)

### PR DESCRIPTION
**Pull request by eduardorcury**
# Pull Request

## What I did

Update on the README file of the *aws apply terraform* command. 
I also changed the config.json file to allow the user to type the repository URL, instead of selecting it (see comments on https://github.com/ZupIT/ritchie-formulas/issues/196)

## How I did it

I searched the documentation to rebuild the formula and test it.

## How to verify it

Run *rit aws apply terraform* and read de README.md.

## What kind of change does this PR make

- [ ] Major
- [x] Minor
- [ ] Patch

## Description for the changelog

Updating aws apply terraform README and changing the repository URL input from "select" to "type".

### CODE REVIEW

File: aws/apply/terraform/README.md [hunk 1]
```diff
@@ -1,7 +1,68 @@
 # Terraform aws apply
 
-## command
+## Premisses
+
+- [Golang installed](https://golang.org/doc/install)
+- [Terraform installed](https://www.terraform.io/downloads.html)
+- Set Github credentials
+- Set AWS credentials
```
Comment (GuillaumeFalourd) [inline aws/apply/terraform/README.md:8]: I believe not all of these points are premisses, as it's (for example) not necessary to have golang install to execute the formula though Docker.

Could you check the formula with Docker execution to update this part ? 😄

Comment (eduardorcury) [reply aws/apply/terraform/README.md:8]: You are right, I will add a note saying that you need Golang for the local run and Docker for the docker run. 
Still, I get this error when trying to run using Docker, any idea why?

image

Comment (GuillaumeFalourd) [reply aws/apply/terraform/README.md:8]: Good point @eduardocurry 💡  
As this is an old formula, it has been implemented without Dockerfile (this Ritchie CLI feature came after).
Therefore, it won't be possible to execute this formula on Docker at the moment.

If you like challenges, you could create another ISSUE to update the formula structure adding this Dockerfile (and maybe other files) following the new structure, which should be the same as the Hello-World formula: https://github.com/ZupIT/ritchie-formulas-demo/tree/master/demo/hello-world 👍

Comment (eduardorcury) [reply aws/apply/terraform/README.md:7]: Ok! I guess we can keep the docker commands in the README file and I will open an issue and try to implement the docker feature :thumbsup:

Comment (GuillaumeFalourd) [reply aws/apply/terraform/README.md:7]: No problem @eduardorcury, thank you 😄

**Comment by GuillaumeFalourd** [comment]
I'll approve and merge this PR as you'll now work on the https://github.com/ZupIT/ritchie-formulas/issues/276 to complete what we discussed here @eduardorcury 👍 

Remeber to update the README file on the other PR according to what you'll implement and turn available.

**Review by GuillaumeFalourd — APPROVED** [review]

**Pull request merged**